src/Cmixin/Holidays/ar-national.php

```diff
@@ -14,12 +14,19 @@
     'substitutes-05-01'                                  => '= 05-01 if Tuesday then previous Monday and if Thursday then next Friday',
     '05-25'                                              => '05-25',
     'substitutes-05-25'                                  => '= 05-25 if Tuesday then previous Monday and if Thursday then next Friday',
+    'guemes-day'                                         => '= 06-17 if Tuesday,Wednesday then previous Monday and if Thursday,Friday then next Monday',
     '06-20'                                              => '06-20',
     'substitutes-06-20'                                  => '= 06-20 if Tuesday then previous Monday and if Thursday then next Friday',
     '07-09'                                              => '07-09',
-    '3rd-monday-of-august'                               => '= third Monday of August',
-    '10-12'                                              => '= 10-12 if Tuesday,Wednesday then previous Monday and if Thursday,Friday,Saturday,Sunday then next Monday',
-    '4th-monday-of-november'                             => '= fourth Monday of November',
+//    '3rd-monday-of-august'                               => '= third Monday of August',
+    'Jose-de-San-Martin'                                 => '= 08-17 if Tuesday,Wednesday then previous Monday and if Thursday,Friday then next Monday',             
+    'day-of-respect-for-cultural-diversity'              => '= 10-12 if Tuesday,Wednesday then previous Monday and if Thursday,Friday then next Monday',
+    'substitutes-day-of-respect-for-cultural-diversity'  => '10-08',    //especial day for 2021
```

Comment:
If it won't be holiday on next years, precise the year:
```suggestion
    'substitutes-day-of-respect-for-cultural-diversity'  => '2021-10-08',    //especial day for 2021
```